README.md

```diff
@@ -1,81 +1,164 @@
 # open_agb_firm
+open_agb_firm is a bare metal interface for *natively* running GBA games and homebrew using the 3DS's built-in GBA hardware.
 
-open_agb_firm is a bare metal app for running GBA games and homebrew using the 3DS builtin GBA hardware.
+open_agb_firm is also a complete and better alternative to GBA VC injects (AGB_FIRM), allowing for:
+* Launching GBA files directly from the SD card
+* Writing save files directly to the SD card
+* Automatic save type configuration using an included database
+* User configuration, such as gamma settings
+* And more to come!
 
 ## Disclaimer
-open_agb_firm is currently in alpha. While open_agb_firm is relatively stable, there are few quirks that have yet to be fixed. See [Known issues](#known-issues) and [Troubleshooting](#troubleshooting) for more information.
+open_agb_firm is currently in alpha. While open_agb_firm is relatively stable and safe to use, some quirks that have not been fixed. See [Known Issues](#known-issues) for more information.
 
 Additionally, we are not responsible for any damage that may occur to your system as a direct or indirect result of you using open_agb_firm.
 
-## How to build
-To compile open_agb_firm, the following needs to be installed:
-* [devkitARM](https://sourceforge.net/projects/devkitpro/)
-* [Corelink DMA-330 assembler](https://github.com/profi200/dma330as)
-* [CTR firm builder](https://github.com/derrekr/ctr_firm_builder)
-
-Additionally, 7-Zip (or on Linux, p7zip) needs to be installed to make release builds. Also make sure the dma330as and CTR firm builder binaries are in the $PATH environment variable and accessible to the Makefile. Build open_agb_firm as a debug build via `make` or as a release build via `make release`.
-
-## How to use
-The process to set up and launch open_agb_firm is similar that of [GodMode9](https://github.com/d0k3/GodMode9).
-* Download the [latest release](https://github.com/profi200/open_agb_firm/releases/latest) and extract it to obtain `open_agb_firm.firm`.
-* Copy the `open_agb_firm.firm` file to your 3DS's SD card at `/luma/payloads` if using Luma3DS or elsewhere if using fastboot3DS.
-* Launch open_agb_firm using Luma3DS by holding START while booting your 3DS or assign the firm to a slot if you're using fastboot3DS.
-* After open_agb_firm launches, use the file browser to navigate to a .GBA ROM to run.
+## Setup
+* Download the [latest release](https://github.com/profi200/open_agb_firm/releases/latest) and extract it.
+* Copy the `open_agb_firm.firm` file to your 3DS's SD card at `/luma/payloads` if you're using Luma3DS or `/gm9/payloads` if you're using fastboot3DS.
```

Comment:
I considered this the "default location" of FIRMs for fastboot3ds since it expects GodMode9 to be stored here, and FIRMs stored here would also be picked up by GodMode9 in its payloads menu. However, I can change this back to "elsewhere" if you want.

Reply:
""default" location of FIRMs for fastboot3ds since it expects GodMode9 to be stored here" Does it? fb3DS doesn't have any default path. When you setup slots you always set the path.

Reply:
Fixed.